string table_name
string topic_name
time start_time
---
bool success
